dll/win32/oleacc/lang/oleacc_De.rc

```diff
@@ -97,7 +97,7 @@ STRINGTABLE
     IDS_STATE_PRESSED           "pressed"
     IDS_STATE_CHECKED           "checked"
     IDS_STATE_MIXED             "mixed"
-    IDS_STATE_READONLY          "read only"
+    IDS_STATE_READONLY          "read-only"
```

Comment:
This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l151